### PR TITLE
Feature/fine grained data map

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

**Comment by geekbeast** [comment]
@katherinebernstein Check out the SearchService changes.

File: src/main/java/com/kryptnostic/datastore/cassandra/RowAdapters.java [hunk 1]
```diff
@@ -137,6 +137,29 @@ private RowAdapters() {
...
+            String entityId,
+            SetMultimap<UUID,ByteBuffer> eb,
+            Map<UUID, PropertyType> authorizedPropertyTypes,
+            ObjectMapper mapper ) {
+        final SetMultimap<UUID, Object> m = HashMultimap.create();
+        eb.entries().stream().forEach( e -> {
+            UUID propertyTypeId = e.getKey();
+            byte[] property = e.getValue().array();
+            PropertyType pt = authorizedPropertyTypes.get( propertyTypeId );
+            if ( propertyTypeId != null ) {
```
Comment (katherinebernstein) [inline src/main/java/com/kryptnostic/datastore/cassandra/RowAdapters.java]: should be `if (pt != null)` ?

**Comment by katherinebernstein** [comment]
@geekbeast the SearchService changes in datastore?

File: src/main/java/com/kryptnostic/datastore/cassandra/RowAdapters.java (hunk 1 above)
Comment (geekbeast) [inline src/main/java/com/kryptnostic/datastore/cassandra/RowAdapters.java]: nice catch

**Comment by geekbeast** [comment]
Yes

**Pull request merged**